### PR TITLE
Label For Toggle

### PR DESCRIPTION
**Pull request by Hari-07**
Fixes #
Fix for issue #213, #166 

#### Describe the changes you have made -
Added the label 'theme' under the toggle

#### Screenshots of the changes (If any) -
2020-05-20 15 43 38 docs circuitverse org 3dcf21c69bde
2020-05-20 15 43 53 127 0 0 1 f37b6d85c0f0

Exisiting Version- Above
My version - Below

### CODE REVIEW

**Comment by nitin10s** [comment]
Please add a screenshot.

File: docs/index.html [hunk 1]
```diff
@@ -59,12 +59,18 @@
...
-				function(hook) 
+				function(hook)
 				{
 
 				      var footer = [
 					'<header>',
-				       	'<span><label class="switch"><input title="Toggle Dark theme" onclick="setStyleSheet();" type="checkbox"><span class="slider round"></span></label></span>',
+				       	'<div>',
+									'<label class="switch">',
+										'<br><p color:> Theme</p>',
```
Comment (mahmodHammad) [inline docs/index.html:69]: <p color:> ! 
why

**Comment by YashKumarVerma** [comment]
@Hari-07  working ?

**Comment by Hari-07** [comment]
Oh I'm sorry I thought it was on hold. I had set it all up back then. I'll put a screenshot as soon as possible

File: docs/index.html (hunk 1 above)
Comment (Hari-07) [reply docs/index.html:69]: I removed that. Tbh i don't exactly remember but i think i was trying to style it

**Pull request closed without merging**